tools/src/versioning/android/transforms/vendoredModulesTransforms.ts

```diff
@@ -48,5 +43,23 @@ export function exponentPackageTransforms(prefix: string): Record<string, String
         replaceWith: `import ${prefix}.$1`,
       },
     ],
+    '@react-native-community/slider': [
+      {
+        find: /\bimport (com\.reactnativecommunity\.slider)/g,
+        replaceWith: `import ${prefix}.$1`,
+      },
+    ],
+    'react-native-gesture-handler': [
+      {
+        find: /\bimport (com.swmansion.gesturehandler)/g,
+        replaceWith: `import ${prefix}.$1`,
+      },
+    ],
+    'react-native-screens': [
+      {
+        find: /\bimport (com.swmansion.rnscreens)/g,
+        replaceWith: `import ${prefix}.$1`,
+      },
+    ],
```

Comment:
versioning import in `ExponentPackage.kt`, because new style vendoring, these modules are not inside `host.exp.exponent.modules` package.